.github/workflows/continous-integration.yml

```diff
@@ -722,6 +733,7 @@ jobs:
     strategy:
       matrix:
         image: [default, full, mitie-en, spacy-de, spacy-it, spacy-en]
+        arch: [linux/amd64, linux/arm64]
```

Comment:
Could we apply this same matrix approach to the section above as well so we're building them in a consistent way?